webapp/channels/src/components/admin_console/access_control/policy_details/policy_details.test.tsx

```diff
@@ -36,6 +36,11 @@ describe('components/admin_console/access_control/policy_details/PolicyDetails',
     const mockGetVisualAST = jest.fn();
     const defaultProps = {
         policyId: 'policy1',
+        accessControlSettings: {
```

Comment:
perhaps add some simple unit tests for positive and negative scenarios for the setting, to validate it shows/hide the shield icon on each scenario?